backend/app/Http/Controllers/Admin/QuestionController.php

```diff
@@ -30,6 +30,26 @@ public function index()
      */
     public function store(Request $request)
     {
-      
+        $categoryRules = [
+            'category_id' => 'required',
+            'value' => 'required'
+        ];
+
+        $this->validate($request, $categoryRules);
+
+        $newQuestion = Question::create($request->all());
+
+        $newChoices[] = '';
+        $choices = $request->input('choices');
+        foreach($choices as $item) 
+        {
+            $newChoices = Choice::create([
+                'question_id' => $newQuestion->id,
+                'value' => $item['value']
+            ]);
+            $newChoices->save();
+        }
```

Comment:
have you checked the insert? so that we will not be calling the create every time. 
https://stackoverflow.com/questions/12702812/bulk-insertion-in-laravel-using-eloquent-orm
for the insert method, a query will only be executed once. if we loop through the data and execute a create statement, there will be a lot of queries